README.md

```diff
@@ -24,3 +24,19 @@ resource "flux_bootstrap_git" "this" {
   path = "clusters/my-cluster"
 }
 ```
+
+## Developer Guide
```

Comment:
Them main readme is not the place for dev docs, we have [CONTRIBUTING.md](https://github.com/fluxcd/terraform-provider-flux/blob/main/CONTRIBUTING.md)